library/dd-sdk-android-session-replay/consumer-rules.pro

```diff
@@ -0,0 +1,5 @@
+# Keep the optional selector class name. We need this in the SR recorder.
+-keepnames class android.widget.DropDownListView
```

Comment:
this is a system class, it won't be obfuscated, so I think this line can be removed.

Reply:
good point ;)